website/content/en/docs/ansible/development-tips.md

```diff
@@ -187,7 +187,7 @@ the below section for Ansible Operator specific annotations.
 #### Ansible Operator annotations
 This is the list of CR annotations which will modify the behavior of the operator:
 
-**ansible.operator-sdk/reconcile-period**: Used to specify the reconciliation
+**ansible.operator-sdk/reconcile-period**: Allow to specify the reconciliation
```

Comment:
Specifies the maximum time before a reconciliation is triggered. Note that at scale, this can reduce performance, see [watches reference][ansible_operator_watches_ref] for more information.

[ansible_operator_watches_ref]: ../reference/watches

Reply:
It does not `Specifies the maximum time before reconciliation is triggered`. See that it will make controller use requeuAfter the period informed.

Reply:
"maximum" because a reconcile will also occur in a shorter amount of time if a watched event comes through.